lib/web_ui/lib/src/engine/util.dart

```diff
@@ -627,6 +628,23 @@ extension JsonExtensions on Map<dynamic, dynamic> {
   }
 }
 
+/// Extracts view ID from the [MethodCall.arguments] map.
+///
+/// Throws if the view ID is not present or if [arguments] is not a map.
```

Comment:
This is going to throw a very ugly error when it attempts to return `null` instead of `int` right? Should this throw a more specific exception when `tryViewId` returns `null`?